<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Amp</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="horizontalSliderAmp">
        <property name="value">
         <number>99</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLCDNumber" name="lcdNumberAmp">
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Freq</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="horizontalSliderFreq">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLCDNumber" name="lcdNumberFreq">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Veloc</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSlider" name="horizontalSliderVeloc">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLCDNumber" name="lcdNumberVeloc">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="segmentStyle">
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="30,70">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditEsq"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCopia">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/resources/morre.png</normaloff>:/resources/morre.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEditDir"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Plotter" name="widgetPlotter" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButtonDenovo">
        <property name="text">
         <string>Morre de novo</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberX"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>y</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lcdNumberY"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonOk">
        <property name="text">
         <string>Morre, diabo!</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbreDialogo"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAbreDialogo"/>
   <addaction name="actionQuit"/>
  </widget>
  <action name="actionAbreDialogo">
   <property name="text">
    <string>AbreDialogo</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/morre.png</normaloff>:/resources/morre.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Plotter</class>
   <extends>QWidget</extends>
   <header>plotter.h</header>
   <container>1</container>
   <slots>
    <signal>mudouX(int)</signal>
    <signal>mudouY(int)</signal>
    <slot>mudaAmplitude(int)</slot>
    <slot>mudaFrequencia(int)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonDenovo</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mataTudo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>460</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAmp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberAmp</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderFreq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberFreq</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderVeloc</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumberVeloc</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderAmp</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>mudaAmplitude(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderFreq</sender>
   <signal>valueChanged(int)</signal>
   <receiver>widgetPlotter</receiver>
   <slot>mudaFrequencia(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbreDialogo</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>abreDialogo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>249</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetPlotter</sender>
   <signal>mudouX(int)</signal>
   <receiver>lcdNumberX</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>371</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widgetPlotter</sender>
   <signal>mudouY(int)</signal>
   <receiver>lcdNumberY</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>391</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>mataTudo()</slot>
  <slot>abreDialogo()</slot>
 </slots>
</ui>
